AppBokerASP/DeviceManager.cs

```diff
@@ -220,14 +232,14 @@ public async Task GetZigbeeDevices()
                         case "lumi.weather": dev = new XiaomiTempSensor(id, client!); break;
                         case "lumi.router": dev = new LumiRouter(id, client!); break;
                         case "L1529":
-                        case "FLOALT panel WS 60x60": dev = new FloaltPanel(id, @$"{Config.HttpUrl}/set/" + deviceRes._id, client!); break;
+                        case "FLOALT panel WS 60x60": dev = new FloaltPanel(id, client!); break;
```

Comment:
As per my request in Teamspeak, dont use client! everywhere, it's better to check it once before the switch case and then return, otherwise this would throw an exception during runtime